### PR TITLE
feat: add project section desktop design

### PR DESCRIPTION
**Pull request by izzett222**
## Describe your changes
Created a Project component. on desktop, created two sections, the first section contains the list of items in H1 style, and the second section contains the project card contains an image and text below the image; the cards will be displayed when the user hovers over the project item, respectively .
 when a user hovers on a project title, underline and arrow is added.
## Issue ticket number and link
id -> 017
link -> [here](https://www.notion.so/apeunit/projects-section-ff9553bcac9647d2ab229a7e2c110782)
## Tasks completed
- [x] add the project section desktop design
- [x] link the project title to website outside the page 
- [x] added underline and arrow effect on hover 
## Tasks not completed
none
## Screenshots (if needed)

### CODE REVIEW

File: components/sections/homepage/Projects.jsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+import ProjectCard from "../../ui/ProjectCard";
+import data from "../../../data/projectCard";
+
+export default function Projects() {
+  const [show, setShow] = useState();
+  const handleHover = (value) => {
+    return function (event) {
+            setShow(value);
+ 
+    };
```
Comment (hirwaaldo1) [inline components/sections/homepage/Projects.jsx:12]: remove the space

File: components/ui/ProjectCard.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import Image from "next/image";
+export default function ProjectCard({src, text, id, show}) {
+  return (
+    <div className={`absolute w-full duration-300 ease-out  transition-all ${ id === show ? "opacity-100 pt-3.5": "opacity-0 pt-3.5"}`}>
+      <div className="relative h-111 w-full drop-shadow-sm rounded-xl">
+        <Image
+        id="img"
```
Comment (hirwaaldo1) [inline components/ui/ProjectCard.jsx]: remove this id because you didn't use it

File: components/ui/ProjectCard.jsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+          src={src}
+          priority
+          className="object-cover"
+          fill
+          alt=""
+        />
+      </div>
+
```
Comment (hirwaaldo1) [inline components/ui/ProjectCard.jsx]: here too remove the space

**Review by hirwaaldo1 — CHANGES_REQUESTED** [review]
change this,it will be great

**Review by hirwaaldo1 — APPROVED** [review]
Well done

**Review by mutesialine — APPROVED** [review]
this looks good

**Pull request merged**